### PR TITLE
#173 Beautify admin home page

### PR DESCRIPTION
**Pull request by denys281**
Added basic styles for admin dashboard
New dashboard with basic charts and numbers
Added chartkick and groupdate gems
Update seeds for Events, Talks
Added seeds for dummy users

@sveredyuk Should I move 'count' and 'group by'  things from views to some other places?

### CODE REVIEW

**Review by sveredyuk — COMMENTED** [review]
@denys281 well dome. Some screenshots ?

**Comment by sveredyuk** [comment]
all things that count, get, query and group data -> to finders

**Comment by denys281** [comment]
@sveredyuk screenshots in #173. Ok, will do it.

File: app/views/admin/home/index.slim [hunk 1]
```diff
@@ -1,4 +1,57 @@
-h2 = 'Welcome, Pivorak!'
+.ui.main.grid
+  h1.ui.header.datetime Welcome, Pivorak!
```
Comment (sveredyuk) [inline app/views/admin/home/index.slim]: remove, please

File: app/assets/stylesheets/admin/styles.scss [hunk 2]
```diff
@@ -10,3 +10,27 @@
 .right-align {
   text-align: right;
 }
+
+/* DASHBOARD STYLES */
+.ui.dashboard.segment {
```
Comment (sveredyuk) [inline app/assets/stylesheets/admin/styles.scss]: extract to `_dashboard.scss` and import it in `application.scss`

File: app/views/admin/home/index.slim [hunk 3]
```diff
@@ -1,4 +1,57 @@
...
 
-.ui.list
-  == render 'layouts/admin/navigation'
+  .doughnut-chart.column
+    .ui.segments
+      .ui.segment
+        h3.ui.blue.header
+          | Events by status
+      .ui.segment
+        = pie_chart Event.group(:status).count
```
Comment (sveredyuk) [inline app/views/admin/home/index.slim]: you can extract all data to the controller helper_method something like `stat`.
```ruby
def stat
  {
    events_by_staus: Event.group(:status).count,
    some_other: ....
  }
end
```

Comment (sveredyuk) [reply app/views/admin/home/index.slim]: let controller to rule queries into db

Comment (sveredyuk) [reply app/views/admin/home/index.slim]: and split this template into small partials

**Comment by sveredyuk** [comment]
- checkout into development branch
- remove it
- pull new development
- rebase your branch with new development
- push your new branch --force

**Comment by denys281** [comment]
@sveredyuk Please, review.

**Review by sveredyuk — APPROVED** [review]

**Pull request merged**